src/components/signupForm/index.tsx

```diff
@@ -66,57 +56,60 @@ const SignupForm: React.FC<SignupFormProps> = ({ onFinish }) => {
               ]}
             >
               <Input placeholder="Last Name" />
-            </Form.Item>
-          </Col>
-        </Row>
-        <Form.Item
-          name="username"
-          rules={[{ required: true, message: 'Please enter a username!' }]}
-        >
-          <Input placeholder="Username" />
-        </Form.Item>
-        <Form.Item
-          name="email"
-          rules={[
-            { required: true, message: 'Please input your email!' },
-            {
-              pattern: /^\S+@\S+\.\S{2,}$/,
-              message: 'Not a valid email address',
-            },
-          ]}
-        >
-          <Input placeholder="Email" />
-        </Form.Item>
-        <Form.Item
-          name="password"
-          rules={[
-            { required: true, message: 'Please enter a password!' },
-            { min: 8, message: 'Password must be at least 8 characters long' },
-          ]}
-        >
-          <Input.Password placeholder="Password" />
-        </Form.Item>
-        <Form.Item
-          name="cpassword"
-          dependencies={['password']}
-          rules={[
-            {
-              required: true,
-              message: 'Please confirm your password!',
-            },
-            ({ getFieldValue }) => ({
-              validator(_, value) {
-                const password = getFieldValue('password');
-                if (value && password !== value) {
-                  return Promise.reject('Passwords do not match');
-                }
-                return Promise.resolve();
+            </FormHalfItem>
+          </FormRow>
+          <Form.Item
+            name="username"
+            rules={[{ required: true, message: 'Please enter a username!' }]}
+          >
+            <Input placeholder="Username" />
+          </Form.Item>
+          <Form.Item
+            name="email"
+            rules={[
+              { required: true, message: 'Please input your email!' },
+              {
+                pattern: /^\S+@\S+\.\S{2,}$/,
```

Comment:
⛏️ for here and anywhere else we have an email input we shouldnt use our own regex, using input type=email would be better.